raiden-dapp/src/locales/en.json

```diff
@@ -875,5 +875,26 @@
         "button": "Cancel"
       }
     }
+  },
+  "quick-pay": {
+    "transfer-information-labels": {
+      "header": "Payment Information",
+      "target-address": "Receiver",
+      "token-amount": "Payment amount"
+    },
+    "action-titles": {
+      "channel-deposit": "Channel Deposit",
+      "channel-open": "Open Channel",
+      "channel-deposit-and-transfer": "Channel Deposit & Pay",
+      "channel-open-and-transfer": "Channel Open & Pay"
+    },
+    "action-messages": {
+      "transfer": "Please confirm the above payment information.",
+      "channel-deposit-and-transfer": "{deposit} will be deposited into the existing channel and {payment} will be paid out.",
+      "channel-open-and-transfer": "{deposit} will be deposited into a new channel and {payment} will be paid out."
+    },
+    "invalid-parameter-error": "Any of the given payment paramter is invalid. Please verify the used link is correct."
```

Comment:
"Some payment parameters are invalid. Please verify that the link is correct."

Reply:
✔️